src/widget/wtracktableview.cpp

```diff
@@ -828,20 +831,49 @@ void WTrackTableView::contextMenuEvent(QContextMenuEvent* event) {
     if (modelHasCapabilities(TrackModel::TRACKMODELCAPS_ADDTOCRATE)) {
         m_pCrateMenu->clear();
         CrateSelectResult allCrates(m_pTrackCollection->crates().selectCrates());
+
+        const QList<TrackId> trackIds = getSelectedTrackIds();
+
+        QHash<CrateId, int> currentCrates(m_pTrackCollection->crates().\
+                                          countTracksInCrates(trackIds));
         Crate crate;
         while (allCrates.populateNext(&crate)) {
-            auto pAction = std::make_unique<QAction>(crate.getName(), m_pCrateMenu);
+            auto pAction = std::make_unique<QWidgetAction>(m_pCrateMenu);
```

Comment:
we have recently introduced "make_parented" to document the qt object tree ownership and to avoid the unused unique_ptr nature.